robot/src/main/java/us/ilite/robot/Robot.java

```diff
@@ -42,11 +47,21 @@
     private DriveController mDriveController = new DriveController(new StrongholdProfile());
     private Drive mDrive = new Drive(mData, mDriveController);
     private DriverInput mDriverInput = new DriverInput(mDrive, mData);
-
+    
     private Trajectory<TimedState<Pose2dWithCurvature>> trajectory;
 
     @Override
     public void robotInit() {
+        // Init static variables and get singleton instances first
```

Comment:
Should `initTimer` be started before these calls?

Reply:
Not necessarily.  Nothing is actually called on the timer yet.